src/styles/scss/_layouts.scss

```
@@ -0,0 +1,62 @@
+/* Content Wrap
+###########################################################################  */
+
+.content-wrap {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  align-items: center;
+  justify-content: center;
+  background-color: $light;
+
+  :nth-child(even) {
```

Ich hab mir den build nicht angeschaut aber da müsste ein `&` fehlen:

```suggestion
  &:nth-child(even) {
```